pkg/sql/catalog/systemschema/system.go

```diff
@@ -247,9 +247,10 @@ CREATE TABLE system.table_statistics (
 	"rowCount"      INT8       NOT NULL,
 	"distinctCount" INT8       NOT NULL,
 	"nullCount"     INT8       NOT NULL,
+  "avgSize"       INT8       NOT NULL,
 	histogram       BYTES,
 	CONSTRAINT "primary" PRIMARY KEY ("tableID", "statisticID"),
-	FAMILY ("tableID", "statisticID", name, "columnIDs", "createdAt", "rowCount", "distinctCount", "nullCount", histogram)
+	FAMILY ("tableID", "statisticID", name, "columnIDs", "createdAt", "rowCount", "distinctCount", "nullCount", "avgSize", histogram)
```

Comment:
This change needs to be accompanied by a tenant migration, otherwise existing clusters, which already have a `table_statistics` table created before this change, will run into trouble when they're upgraded to a version which does include this change. I suspect that several CI failures are being caused by this missing migration.

Doing so involves adding a new internal version and key to `cockroach_versions.go` and binding a migration to it via a call to `migration.NewTenantMigration`. A good source of inspiration for how to do this and test it is `alter_web_sessions_create_indexes.go` which implements a similar migration.